shipyard-controller/docs/swagger.yaml

```diff
@@ -778,7 +768,7 @@ paths:
           description: Bad Request
           schema:
             $ref: '#/definitions/models.Error'
-        "403":
```

Comment:
is the code returning a 403 or 404?

Reply:
`404` https://github.com/keptn/keptn/blob/9915466adbb639dd40688791b23e62b86f2469f9/shipyard-controller/handler/projecthandler.go#L204

Reply:
The code returns 404: https://github.com/keptn/keptn/blob/master/shipyard-controller/handler/projecthandler.go#L204